Add Support for Config

Ririio opened the pull request:
I added an "input" folder to ensure that we can test files in one singular directory as to having to add them manually each time. I also have the "ssg-config.json" located at the bottom of the file for when the user decides to run a script.

Updated the README to add the "config" in the table.

I can readjust if you happen to have any problems with the code

SerpentBytes commented on src/main.js:
```diff
@@ -12,6 +12,8 @@ module.exports.main = (args) => {
     dispatch("-h");
   } else if (args[0] === "-i" || args[0] === "--input") {
     dispatch("-i", args);
+  } else if (args[0] === "-c" || args[0] === "--input") {
```
There's a typo here.

Ririio replied:
I thought I fixed this already weird

Ririio replied:
It should be fixed now

SerpentBytes reviewed (changes requested):
replace "--input" with "--config"

Boldgainobu reviewed (approved):
https://github.com/SerpentBytes/ririio-ssg

Boldgainobu reviewed (commented):
https://github.com/Ririio/siteit/tree/issue-20

SerpentBytes commented:
1. Using config flag and passing a file name that ends with an invalid extension prints testing `console.log`  and multiple error messages.

I entered the following: 
```
siteit --config ../input.md
```

Terminal output: 
```
-- Invalid or multiple source supplied --
-c
-- ERROR --
siteit -c myfile.json
usage: siteit --config myfile.json
```
2. Passing a valid file does not generate status and success messages like functions responsible for parsing `.txt` and `.json` files.
3. Passing an input file with the following `.json` config file renders inaccurate HTML. See below:

```
{
  "input" : "./inputs"
}
```

Output in terminal :

```
-- Attempting to create output directory --
-- Output directory exists. Attempting to delete it --
-- Pre-existing output directory deleted --
-- DONE creating output directory: /dist --
-- input.html generated in dist directory --

```

HTML rendered:

```
<!DOCTYPE html>
<html lang="en">
  <head>
    <meta charset="utf-8">
    <link rel="stylesheet" type="text/css" href="../src/siteit.css" />
    <link rel="preconnect" href="https://fonts.googleapis.com">
    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
    <link href="https://fonts.googleapis.com/css2?family=Questrial&display=swap" rel="stylesheet">
    <title>input</title>
    <meta name="viewport" content="width=device-width, initial-scale=1">
  </head>
  <body>
    <p>{ "input" : "./inputs" }</p>
  </body>
</html>
```

### Expected Behaviour
 When a valid `config` file is passed to the tool, it should create an HTML version of all files inside the `./inputs` directory.

Ririio commented:
Problem 1: I fixed the issue regarding that
Problem 2: I added a success status message
Problem 3: That's not really for my config file, I tested your input and you don't go through multi-layer directories even with it 

**Running the command**
![image](https://user-images.githubusercontent.com/32577022/193964200-9bc49cc2-9f96-484a-8f8e-36e3cdadba15.png)

**HTML contents**
![image](https://user-images.githubusercontent.com/32577022/193964263-67bbfcb4-8ea7-4c93-9516-3335cc619cf1.png)

**Web Output**
![image](https://user-images.githubusercontent.com/32577022/193964323-b6c668bf-2cd3-4cac-9733-2393e70c104c.png)

**Clicking the link**
![image](https://user-images.githubusercontent.com/32577022/193964360-fdef0d5b-28ab-4bcc-b0de-645450d7047a.png)

The .json config file should have 

`{
  "input": "./inputs/Sherlock-Holmes-Selected-Stories"
}`

for it specifies the directory and its contents
